Week5/solution/Week5_Solution.py

```diff
@@ -0,0 +1,33 @@
+# INSERTING RECORDS
+def insertFunct():
```

Comment:
why u use the insertFunct and what is the work of it ??

Reply:
its a user defined function, this function is used to insert records